localisation/00_wc_artifact_localisation.csv

```diff
@@ -178,3 +178,35 @@ tauren_totem;Tauren Totem;;Taurentotem;;;;;;;;;;;x
+whispers_stewardship_2;Visions of Black Empire;;Visionen des schwarzen Reiches;;;;;;;;;;;x
+whispers_stewardship_3;Visions of Black Empire;;Visionen des schwarzen Reiches;;;;;;;;;;;x
+whispers_stewardship_4;Visions of Black Empire;;Visionen des schwarzen Reiches;;;;;;;;;;;x
+whispers_intrigue_1;Murmurs of Deep Below;;Raunen aus der Tiefe;;;;;;;;;;;x
+whispers_intrigue_2;Murmurs of Deep Below;;Raunen aus der Tiefe;;;;;;;;;;;x
+whispers_intrigue_3;Murmurs of Deep Below;;Raunen aus der Tiefe;;;;;;;;;;;x
+whispers_intrigue_4;Murmurs of Deep Below;;Raunen aus der Tiefe;;;;;;;;;;;x
+whispers_learning_1;Augurs of Eldritch Insights;;Offenbarungen der unheimlichen Einblicke;;;;;;;;;;;x
+whispers_learning_2;Augurs of Eldritch Insights;;Offenbarungen der unheimlichen Einblicke;;;;;;;;;;;x
+whispers_learning_3;Augurs of Eldritch Insights;;Offenbarungen der unheimlichen Einblicke;;;;;;;;;;;x
+whispers_learning_4;Augurs of Eldritch Insights;;Offenbarungen der unheimlichen Einblicke;;;;;;;;;;;x
+whispers_martial_1_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_martial_2_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_martial_3_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_martial_4_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_stewardship_1_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_stewardship_2_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_stewardship_3_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_stewardship_4_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_intrigue_1_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_intrigue_2_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_intrigue_3_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_intrigue_4_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_learning_1_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_learning_2_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_learning_3_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
+whispers_learning_4_desc;When someone holds the book, dark whispers creep into the corner of their mind...;;;;;;;;;;;;;x
```

Comment:
"their" is plural, so it would have to be "their minds", I think

Reply:
its singular if the gender is unknown

Reply:
https://www.merriam-webster.com/dictionary/their

Reply:
gender doesn't make sense here. "Their" is plural, so mind->minds

Reply:
Alright, I yield.